barretenberg/cpp/src/barretenberg/bb/main.cpp

```diff
@@ -535,22 +535,44 @@ int main(int argc, char* argv[])
             std::string output_path = get_option(args, "-o", vk_path + "_fields.json");
             vk_as_fields(vk_path, output_path);
         } else if (command == "avm_prove") {
```

Comment:
I think the rest of the commands put the code inside a function. It would be good to do that in the avm commands to keep the switch clean.